app/controllers/restaurants_controller.rb

```diff
@@ -47,11 +47,14 @@ def check_for_exisiting_restaurant
     end
   end
 
+  def restaurant_categories
+    @categories = ['Pizza', 'Sushi', 'Italian', 'Thai', 'Fast Food', 'BBQ', 'French', 'Traditional', 'Vegan', 'Seafood', 'Texmex']
```

Comment:
This feels like something that should be defined in your Restaurant model instead as a constant.
e.g: 
```ruby
CATEGORIES = ['Pizza', 'Sushi', 'Italian', 'Thai' ... ]
```
Then you can access it in your view like `Restaurant::CATEGORIES`

How do you prefer this handle this @tochman